frontend/src/components/creator-modal/creator.tsx

```diff
@@ -108,10 +114,18 @@ const Creator = (): ReactElement => {
     <Modal>
       <form className="create-question-module" onSubmit={saveQuestion}>
         <div className="creator-header">
-          <button className="exit" onClick={closePreviewQuestion}>
+          <button
+            className="exit"
+            onClick={(event: SyntheticEvent): void => {
```

Comment:
Remove the button fix this isn't the way

Reply:
Why?

Reply:
This is a hacky solution and doesn't solve the underlying issues with the original approach

Reply:
The fix is on my other branch

Reply:
It is normal to prevent default button behavior in forms, though. I can extract this to all be in one function instead of relying on lambdas, but I don't see how this is a hack or that there actually are any underlying issues

Reply:
Yes when you are trying to prevent default behavior the issue here is 

You guys set ￼implicitly created the button to react the way that it does because buttons by default have type set to submit

The best here is to set an attribute type to value button

Reply:
Ah ok. I can switch to using that then